connectors/sources/mysql.py

```diff
@@ -243,8 +243,7 @@ async def _connect(self, query, fetch_many=False, **query_kwargs):
 
         query_kwargs["database"] = self.database
         formatted_query = query.format(**query_kwargs)
-        size = int(self.configuration.get("fetch_size", DEFAULT_FETCH_SIZE))
-
+        size = self.configuration["fetch_size"]
```

Comment:
It removes `DEFAULT_FETCH_SIZE`. What if there's no key `fetch_size` in `self.configuration`